.github/workflows/release-update-cloudfoundry-index.yml

```diff
@@ -0,0 +1,37 @@
+# This action updates the CloudFoundry java-buildpack release index after each release.
+# See https://github.com/cloudfoundry/java-buildpack/blob/main/docs/extending-repositories.md
+# Prerequisite: the repo must have a branch named "cloudfoundry".
+
+name: Update cloudfoundry release index
+on:
+  release:
+    types: [released]
+  workflow_dispatch:
+
+permissions:
+  contents: write
+
+jobs:
+
+  update-cloudfoundry-index-yml:
+    runs-on: ubuntu-latest
+    outputs:
+      version: ${{ steps.create-github-release.outputs.version }}
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          ref: 'cloudfoundry'
+
+      - run: sudo apt-get install jq python3-pip
+      - run: pip install yq
+
+      - run: |
+          wget https://repo1.maven.org/maven2/io/opentelemetry/javaagent/opentelemetry-javaagent/maven-metadata.xml
```

Comment:
That makes sense, though I'd be concerned about linking to a release that hasn't been created yet.
This should be ok though as it would just mean that releases are delayed.  If someone does ask for a current release, the job can be triggered manually.

Reply:
Another option... we could add a chron schedule to trigger this action 1-2 weeks after we normally do the release. This would still result in a delay, but not as significant.

Reply:
I agree that we should do this on a schedule, maybe once a day.  It would be great if it could check the latest actually available on sonatype/central against the latest in the CF file, and only update when it's behind.

Reply:
> I agree that we should do this on a schedule, maybe once a day.

Yeah, I also think that's probably the best option. AFAIK the usual maven delay is <24h, so it this should cover all the bases.

Reply:
Ok, changed it to run daily and updated the script to avoid error if no changes were made.